### PR TITLE
Updating "Full metrics pipeline" para(resource-usage-monitoring.md Modified)

### PR DESCRIPTION
**Pull request by ShivangShandilya**
Fix: https://github.com/kubernetes/website/issues/31483

The following changes have been made looking at @sftim https://github.com/kubernetes/website/issues/31483#issuecomment-1148521363 to the following issue.

In the section "Full metrics pipeline" it says:

"Prometheus, a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself. Full metrics pipeline projects that are not part of the CNCF are outside the scope of Kubernetes documentation."

The above paragraph was not getting interpreted correctly and was not understandable for a beginner. This PR solves this issue by modifying the para into a more clear one for beginners to comprehend it correctly. Beginner-friendly language is used in this modified para that makes everything really simple to understand.

### CODE REVIEW

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (tengqm) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:65]: Please wrap the long lines manually when appropriate.
You can check how other paragraphs are formatted in markdown.

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (tengqm) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:62]: two dots at the end?

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (tengqm) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:64]: upto is not a valid word?

"glance over at ..."?

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:75]: Resolved

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: Resolved

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: upto was a typo error which I have corrected to - up to
and I think glace over at CNCF landscape is correct, do tell if I'm wrong.
Regards

**Comment by ShivangShandilya** [comment]
All the issues have been corrected by me, please review and merge @tengqm 
Regards and Thank you for your guidance

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:68]:
```suggestion
If you glance over at 
```

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:70]:
```suggestion
you can observe that Kubernetes project doesn't provide a recommendation 
```

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:71]:
```suggestion
of any projects to work on, as it is totally up to the users to find the 
```

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:72]:
```suggestion
project that suites them the best. If someone is to make a full metrics pipeline,
```

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]
@ShivangShandilya PTAL at the above suggestions.

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:70]: I think this one is correct, isn't it @NitishKumar06 . it says if you glance over at CNCF Landscape you can observe that Kubernetes------- if I add It is important to mention it won't make sense with the previous sentence.
Do correct me if I'm wrong.
Regards

**Comment by ShivangShandilya** [comment]
> @ShivangShandilya PTAL at the above suggestions.

Some of the suggestions have been implemented, please review and merge

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: According to style guides, we prefer using _you_ rather than _we_.

Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: I have changed 'we' to 'you' wherever it was used and yes it does look more readable now. Thanks for the suggestion 😁

Comment (NitishKumar06) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: I'm not sure what text in the docs helps in **_observing_** that Kubernetes doesn't provide a recommendation......
I'd still go with the text _It is important to mention that......_

Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: So how to add the CNCF landscape line with It is imp to mention any suggestions? Maybe we can write this Glancing over at CNCF Landscape, It is imp to mention----- is this good??

Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: While you glance over at CNCF Landscape, It is important to mention that the Kubernetes project doesn't provide recommendations.
I think this works

Comment (NitishKumar06) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md]: sounds okayish! Though I'd not provide a lgtm tag until this PR goes through another set of review by other reviewers & maintainers.

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:70]:
```suggestion
you can observe that the Kubernetes project doesn't provide a recommendation 
```
just a quick question or more like a doubt, how can you say that k8s don't provide a recommendation of a set of projects **just by looking at the landscape**? There is no set of lines that gives this kind of proof. Maybe we can modify this!

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:72]:
```suggestion
project that suites them the best. If someone is to make a full metrics pipeline,
```
How about providing a reference link to _full metrics pipeline_?

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]
PTAL.

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:70]: Maybe because when we look at CNCF landscape there are tons of tools to work with k8s, not only Prometheus. One can check out other tools and work with one they think suits them best.

Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:70]: image

Like if I'm looking over at Monitoring, I can see there are many projects other than Prometheus, and one can check out these and work with the one they think suits them best like Open Metrics or Thanos for example

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (reylejano) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:62]:
```suggestion
Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/), 
```
I don't think this statement is correct
/hold

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (sftim) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:74]: I'd write:
```suggestion
you can see a number of monitoring projects that can work with Kubernetes by _scraping_
metric data and using that to help you observe your cluster. It is up to you to select the tool
or tools that suit your needs. The CNCF landscape for observability and analytics includes a
mix of open-source software, paid-for software-as-a-service, and other commercial products.

When you design and implement a full metrics pipeline you can make that monitoring data
available back to Kubernetes. For example, a HorizontalPodAutoscaler can use the processed
metrics to work out how many Pods to run for a component of your workload.
```

Comment (ShivangShandilya) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:74]: Ok, this sounds way better than mine 😂. @NitishKumar06 what's your opinion on this??

**Comment by ShivangShandilya** [comment]
> @ShivangShandilya Ping!
I completely forgot about this PR, lets work on this ASAP and if possible merge it too

**Comment by sftim** [comment]
/label tide/merge-method-squash

Please rebase against _main_ if feasible.

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (sftim) [inline content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:69]: (nit)
```suggestion
If you glance over at the
```

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (sftim) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:75]: I would go with that suggestion @ShivangShandilya (I'm writing as a SIG Docs reviewer).

**Comment by ShivangShandilya** [comment]
So then do I still need to make any changes??

File: content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md
Comment (NitishKumar06) [reply content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md:75]: Proceed with the suggestions given by @sftim

**Comment by NitishKumar06** [comment]
Failing checks. Try passing an empty commit.

**Comment by NitishKumar06** [comment]
I think this PR is ready to proceed with the next set of reviews. Please squash the commits.

**Comment by NitishKumar06** [comment]
If you're unable to squash the commits, try doing this:

1. `git logs` to see the commits.
2. `git rebase -i <commit-hash>` (the commit hash will be of the commit just after the commit you'd like to squash)
3. Sqush the commits by `pick` or `squash`.
4. Force push using `-f`: `git push -f origin <local branch>`

**Comment by ShivangShandilya** [comment]
> If you're unable to squash the commits, try doing this:
> 
> 1. `git logs` to see the commits.
> 2. `git rebase -i <commit-hash>` (the commit hash will be of the commit just after the commit you'd like to squash)
> 3. Sqush the commits by `pick` or `squash`.
> 4. Force push using `-f`: `git push -f origin <local branch>`

I haven't tried squashing commits before and I'm not able to wrap my head around it, even removed the lgtm tag in the 
process and I'm still not able to do it 😥😥

**Comment by kbhawkey** [comment]
Preview:
https://deploy-preview-38816--kubernetes-io-main-staging.netlify.app/docs/tasks/debug/debug-cluster/resource-usage-monitoring/#full-metrics-pipeline

**Pull request merged**
